versioned_docs/version-2.6/how-to-guides/new-user-guides/manage-clusters/create-kubernetes-persistent-storage/manage-persistent-storage/set-up-existing-storage.md

```diff
@@ -14,7 +14,7 @@ To set up storage, follow these steps:
 
 1. [Set up persistent storage.](#1-set-up-persistent-storage)
 2. [Add a PersistentVolume that refers to the persistent storage.](#2-add-a-persistentvolume-that-refers-to-the-persistent-storage)
-3. [Use the PersistentVolume for Pods Deployed with a StatefulSet.](#3-use-the-persistentvolume-for-pods-deployed-with-a-statefulset)
+3. [Use the PersistentVolume for Pods Deployed with a StatefulSet.](#3-use-the-storage-class-for-pods-deployed-with-a-statefulset)
```

Comment:
```suggestion
3. [Use the Storage Class for Pods Deployed with a StatefulSet.](#3-use-the-storage-class-for-pods-deployed-with-a-statefulset)
```

Apologies if I missed the title change in other links to this page

Reply:
Fixed in other instance where this change is applicable.